libs/fn/src/lib/select/select.component.html

```diff
@@ -18,8 +18,8 @@
         [cdkConnectedOverlayOrigin]="trigger"
         [cdkConnectedOverlayOpen]="opened && !_optionsListEmpty"
     >
-        <ul [style.width.px]="_selectWidth" aria-label="Select menu" fnSelectMenu #menu>
+        <div [style.width.px]="_selectWidth" aria-label="Select menu" fnSelectMenu #menu>
```

Comment:
Assuming the original syntax for the options (using an `li`) can be used, I think we would want to keep this as a `ul`

Reply:
I think it is more important that the rendered template has a `ul` with direct `li` descendants, otherwise we have invalid HTML. Accessibility tools need this `ul` `li` relationship to work properly.

Reply:
@shrvr What you can do to keep the markup intact is to use ContentChildren for the fn-option, and iterate over it inside ul tag and creating own li elements

Reply:
@mikerodonnell89 we may have issues with both options:
if we keep li[fn-option], then we'll have invalid markup in the template
If we have fn-option, we'll introduce a breaking change, yet, markup will be valid.